vpr/test/test_vpr.cpp

```diff
@@ -188,21 +188,21 @@ TEST_CASE("read_rr_graph_metadata", "[vpr]") {
     std::mt19937 g(1);
     std::shuffle(src_order.begin(), src_order.end(), g);
 
-    CHECK(device_ctx.rr_node_metadata.size() == 1);
-    CHECK(device_ctx.rr_edge_metadata.size() == 1);
+    CHECK(device_ctx.rr_graph_builder.rr_node_metadata_size() == 1);
+    CHECK(device_ctx.rr_graph_builder.rr_edge_metadata_size() == 1);
 
     auto node = arch.strings.intern_string(vtr::string_view("node"));
     auto edge = arch.strings.intern_string(vtr::string_view("edge"));
 
-    for (const auto& node_meta : device_ctx.rr_node_metadata) {
+    for (const auto& node_meta : device_ctx.rr_graph_builder.rr_node_metadata_) {
```

Comment:
@tangxifan,
I was trying to pass the method for calling  rr_node_metadata_. The  rr_node_metadata() is built in rr_graph_builder but I am unable to pass it as every time it throws an error of invalid arguments. When I checked about its declaration it shows that here it is calling rr_node_metadata.origin. Passing directly the .origin also don't work.
Please guide me about how to resolve this issue.